test/setup-ca.sh

```diff
@@ -16,10 +16,16 @@ if [ $cfssl_found -eq 0 ]; then
     exit 1
 fi
 
+CADIR=$(dirname ${CA})
+if ! [ -d ${CADIR} ]; then
+  mkdir ${CADIR}
+fi
```

Comment:
These three lines can be replaced with `mkdir -p "${CADIR}"`.

Reply:
True, fixed.